packages-ts/gauntlet-terra-contracts/src/commands/tooling/updateContractAdmin.ts

```diff
@@ -0,0 +1,95 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { RDD, TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgUpdateContractAdmin } from '@terra-money/terra.js'
+import { CATEGORIES } from '../../lib/constants'
+
+type CommandInput = {
+  newAdmin: string
+  contract: string
+  force?: boolean
+}
+
+export default class UpdateContractAdmin extends TerraCommand {
```

Comment:
This command has nothing to do with our contracts. We can update contract admin for any contract, not just Chainlink contracts.

This should be moved to `gauntlet-terra` base package, so either try to do that now or add an inline comment noting the next steps oh how to restructure this.